velox/vector/DecodedVector.h

```diff
@@ -214,9 +267,13 @@ class DecodedVector {
     BufferPtr nulls;
   };
 
-  /// Returns 'indices' and 'nulls' buffers that represnt the combined
+  /// Returns 'indices' and 'nulls' buffers that represent the combined
   /// dictionary wrapping of the decoded vector. Requires
   /// isIdentityMapping() == false and isConstantMapping() == false.
+  /// NOTE: The nulls buffer returned will also have nulls from the base()
+  /// combined into it. To control which levels are combined, please make sure
+  /// to use makeIndices() instead of decoded() when initializing the
```

Comment:
makeIndices() indices only combines the number of levels its asked to. So if the number of levels also include the base layer, it would combine nulls from it as well. I will update the method comment for makeIndicies and add the same example I added to the class comment to further clarify its usage.